### PR TITLE
Adding a rejection sampled normal distribution function with a modifi…

### PR DESCRIPTION
**Pull request by madebyjeffrey**
Adding a rejection sampled normal distribution function with a modified lower bound values.

### CODE REVIEW

File: cs/random.go [hunk 1]
```diff
@@ -15,3 +19,48 @@ type rng interface {
...
+var initialized = false
+
+func normalDistribution(μ float64, σ float64, x float64) float64 {
+	var eNum = -math.Pow(x-μ, 2)
+	var eDen = 2 * math.Pow(σ, 2)
+
+	return math.Exp(eNum/eDen) / math.Sqrt(2*math.Pi*math.Pow(σ, 2))
+}
+
+func initRandom() {
```
Comment (sirgwain) [inline cs/random.go]: is this something that only happens once a program or is it unique per game? It looks once a program...

If so, golang has an init function you can put in a package that runs once during startup or load (not sure exactly when).

https://go.dev/doc/effective_go#init

**Comment by madebyjeffrey** [comment]
Updated the code to not require any storage. It is simpler. But the limiter code should be removed otherwise we have too many low end concentrations.

File: cs/planet.go [hunk 2]
```diff
@@ -245,11 +244,21 @@ func (p *Planet) randomize(rules *Rules) {
...
-	// create minconc from min to max (31 to 121)
+	// These two variables are the shape of the normal distribution
+	// based on comparing it with Stars! output
+	mean := 80.0
+	variance := 20.0
+
+	// These two are the min and max of the minerals to be returned,
+	// They clamp the results
+	mMin := 1
+	mMax := 126
```
Comment (sirgwain) [inline cs/planet.go]: can we set these to `rules.MinStartingMineralConcentration` and `rules.MaxStartingMineralConcentration`

**Review by sirgwain — APPROVED** [review]
I think this needs some thread checking on it, or it needs to use

**Pull request merged**